src/main/java/teammates/common/util/Const.java

```diff
@@ -259,7 +263,8 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String STUDENT_COURSE_PROFILE = "Your profile in this course";
         public static final String STUDENT_COURSE_DETAILS = "View and edit information regarding your team";
     
-        public static final String STUDENT_FEEDBACK_SESSION_STATUS_AWAITING = "The session is not open for submission at this time. It is expected to open later.";
+        public static final String STUDENT_FEEDBACK_SESSION_STATUS_AWAITING =
+                "The session is not open for submission at this time. It is expected to open later.";
```

Comment:
You just gave me an idea to write another rule to check things getting squished to the right :p